docs/clients/rust/getting_started.rst

```diff
@@ -0,0 +1,138 @@
+.. _ref_rust_getting_started:
+
+Getting started
+---------------
+
+From examples repo
+------------------
+
+If you just want a working repo to get started, clone the Rust client 
+`examples repo`_, type ``edgedb project init`` to start an EdgeDB
+project, and then ``cargo run`` to run the samples.
+
+This tutorial contains a lot of similar examples to those found in the
+``main.rs`` file inside that repo.
+
+From scratch
+------------
+
+The minimum to add to your Cargo.toml to use the client is `edgedb-tokio`_:
+
+.. code-block::
```

Comment:
Just recently learned this language is available!

```suggestion
.. code-block:: toml
```

Reply:
Nice!